packages/cli/lib/services/compile.service.ts

```diff
@@ -31,6 +31,7 @@ class CompileService {
         }
 
         const compiler = tsNode.create({
+            skipProject: true, // when installed locally we don't want ts-node to picks up the package tsconfig
```

Comment:
By default ts-node try to load `tsconfig.json` file it can find within the project. It is not a problem when installing the CLI from npm but when installed locally the project files (including the tsconfig.json files) are still around